backend/app/models/launched_study.rb

```diff
@@ -30,6 +30,18 @@ def completed!
     update!(completed_at: Time.now)
     completed = LaunchedStudy.where(study_id:).complete.count
     Study.update(study_id, completed_count: completed)
+
+    total_completed_for_user = LaunchedStudy.where(user_id:).count
+
+    user_info = UserInfo.for_uuid(user_id)
+    recipient = Struct.new(:email_address, :full_name).new(
+      user_info['email_address'] || 'Admin-Uno@test.openstax.org',
+      user_info[:full_name]
+    )
```

Comment:
Technically this can be moved below the `return unless` since we don't use it if that condition is met 🤔 not a big deal at all though